packages/client/src/main.js

```diff
@@ -1,9 +1,12 @@
 import { createPinia } from 'pinia'
 import { createApp } from 'vue'
+
 import App from './App.vue'
 import router from './router'
 
 import 'bootstrap'
+import 'bootstrap/dist/css/bootstrap.css'
+import './assets/styles/main.css'
```

Comment:
Any particular reason why we import these stylesheets here, and not import the bootstrap styling into the main stylesheet or not link the main stylesheet via the index.html?

Reply:
no particular reason. I can move it to index.html